components/movementsensor/imuvectornav/imu.go

```diff
@@ -72,12 +67,12 @@ type vectornav struct {
 
 	mu      sync.Mutex
 	spiMu   sync.Mutex
-	polling int
+	polling uint
```

Comment:
why this change?

Reply:
`compensateAccelBias` and `compensateDVBias` expect the secondary argument to be of type `uint`. This way we do not have to cast them when the function is called and can instead simply do `v.polling`

Reply:
I see, this is a good change we could even go earlier and cast it in the config

Reply:
oo nice! I will do this.